main.go

```diff
@@ -300,7 +301,11 @@ func CreateExecutionPlan(cfg *Config) (Plan, error) {
 					}
 				}
 
-				envStr := "^:|:^" + strings.Join(e, ":|:")
+				envsDelimiter := ":|:"
+				if f.EnvironmentDelimiter != "" {
```

Comment:
How about checking and setting the delimiter up here: https://github.com/oliver006/drone-gcf/blob/58f0568a73bdd522a0d5d10cac36b51c501af61a/main.go#L137

and then just use it here?